english/fill-sock/fill-sock.md

```diff
@@ -2,4 +2,6 @@ You quickly realise that marshmallows melt faster inside pink socks.
 
 This makes you decide to fill the sock with [more marshmallows](fill-sock.md).
 
-You decide [there's enough marshmallows for now](../marshmallow.md)
\ No newline at end of file
+You decide [there's enough marshmallows for now](../marshmallow.md)
+
+You feel hungry all of a sudden and decide you will [eat the rest of the marshmellows](../eating-walls/eating-marshmellows.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.